meson.build

```diff
@@ -0,0 +1,78 @@
+project('io', 'd', default_options: ['optimization=3'], version: '2.3.3', license: 'BSL-1.0' )
```

Comment:
I'm not at all familiar with meson. What is the purpose of the version number here? We are at 0.2.3. Does this version number have to be updated with every release? If so, can it be automated?

Reply:
Indeed, I will fix this